trace_viewer/extras/importer/trace_event_importer.html

```diff
@@ -406,6 +409,22 @@
       this.model_.samples.push(sample);
     },
 
+    processMemoryDumpEvent: function(event) {
+      if (event.id === undefined) {
+        this.model_.importWarning({
+          type: 'memory_dump_parse_error',
+          message: 'd phase event without a dump ID.'
+        });
+        return;
+      }
+
+      if (!(event.id in this.allMemoryDumpEvents_)) {
+        this.allMemoryDumpEvents_[event.id] = [];
```

Comment:
Can't remember, does JS support

this.allMemoryDumpEvents_[event.id] ||= [];

Or, is that a rubyism?